src/auth/project-config.ts

```diff
@@ -57,6 +62,15 @@ export class ProjectConfig {
    * This is based on the calling code of the destination phone number.
    */
   public readonly smsRegionConfig?: SmsRegionConfig;
+  private readonly multiFactorConfig_?: MultiFactorConfig;
```

Comment:
don't you need multiFactorConfig in ProjectConfigServerResponse and ClientRequest?